### PR TITLE
Fix the script of generating release

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/gen-release.sh [hunk 1]
```diff
@@ -24,8 +24,8 @@ done
 echo "Prepare release for version $1"
 
 # Update release file
-RELEASE_FILE=release/RELEASE
-echo -e "version: $1" > $RELEASE_FILE
+RELEASE_FILE=RELEASE
+sed -i '' "s/tag:.*/tag: $1/" $RELEASE_FILE
```
Comment (khanhtc1202) [inline hack/gen-release.sh:28]: I know we don't have a convention for this but just think it's better to unify with other places we cut this value, so
```bash
head -n 1 RELEASE | cut -d ' ' -f 2
```
(as we did in Makefile)

Comment (knanao) [reply hack/gen-release.sh:28]: I see. Let me replace it with this 🙌

Comment (knanao) [reply hack/gen-release.sh:28]: @khanhtc1202 
Oops, sorry let me check one point.
This script updates the tag in the RELEASE file hence I think this args will be required.

Comment (khanhtc1202) [reply hack/gen-release.sh:28]: ah got that, it's my miss 🙏

**Review by khanhtc1202 — APPROVED** [review]
Nice catch

**Review by nghialv — APPROVED** [review]
Nice catch!

**Pull request merged**